signing/src/main/java/tech/pegasys/web3signer/signing/config/AzureKeyVaultParameters.java

```diff
@@ -12,6 +12,9 @@
  */
 package tech.pegasys.web3signer.signing.config;
 
+import java.util.Collections;
+import java.util.Map;
+
 public interface AzureKeyVaultParameters {
```

Comment:
The AwsKeySigningMetadata doesn't have an implementation so we won't be able to use tags in the key config files

Reply:
tags only make sense for bulk load. For key config file, we always specify the "secret name" to load.